btr-api/src/btr_api/resources/submission.py

```diff
@@ -90,31 +89,45 @@ def latest_submission_for_entity(business_identifier: str):
 
 
 @bp.route("/", methods=("POST",))
+@cross_origin(origin='*')
+@jwt.requires_auth
 def create_register():
     """
     Create a new register.
 
     Returns:
         A tuple containing the response JSON and the HTTP status code.
     """
-    schema_name = "btr-filing.schema.json"
-    schema_service = SchemaService()
-    # schema = schema_service.get_schema(schema_name)
-    # if not schema:
-    #     return {}, HTTPStatus.INTERNAL_SERVER_ERROR
-
     try:
-        if json_input := request.get_json():
-            # normally do some validation here
-            [valid, errors] = schema_service.validate(schema_name, json_input)
-            if not valid:
-                return {"errors": errors}, HTTPStatus.BAD_REQUEST
-
-            submission = SubmissionService.create_submission(json_input)
-            submission.save()
-            return jsonify(id=submission.id), HTTPStatus.CREATED
-
-        return {}, HTTPStatus.BAD_REQUEST
+        # TODO: check auth / validate user access
```

Comment:
Maybe a ticket for this one ? Looks important :D

Reply:
https://github.com/bcgov/entity/issues/19212